awx/main/models/organization.py

```diff
@@ -19,12 +20,14 @@
     ROLE_SINGLETON_SYSTEM_ADMINISTRATOR,
     ROLE_SINGLETON_SYSTEM_AUDITOR,
 )
-from awx.main.models.mixins import ResourceMixin, CustomVirtualEnvMixin
+from awx.main.models.unified_jobs import UnifiedJob
+from awx.main.models.ha import ACTIVE_STATES
```

Comment:
The original definition of ACTIVE_STATES is in unified_jobs IIRC, importing it from other locations that imported it is creating some import spaghetti